### DISCUSSION TITLE
How to always display crosshair at last series in the Candlestick chart

### DISCUSSION THREAD

**Discussion by vnanhtuan** [apexcharts/apexcharts.js#3229] [Q&A]
Hi, 
I want the crosshair always display at the last series in the Candlestick chart. How to do that?
This is my expect:
<img width="711" alt="Screen Shot 2022-06-23 at 20 42 50" src="https://user-images.githubusercontent.com/10513517/175313858-b150c7c8-49e8-4763-ad43-f75cd42f77cb.png">

Thank you.

**Comment by zee0786** [comment]
Im looking exactly the same thing. need to show the crosshair only for the last series data
